pkg/token/token.go

```diff
@@ -514,6 +515,7 @@ func (v tokenVerifier) Verify(token string) (*Identity, error) {
 		if urlErr, ok := err.(*url.Error); ok {
 			return nil, NewSTSError(fmt.Sprintf("error during GET: %v", urlErr.Err))
```

Comment:
why not inc here?

Reply:
right..should do here too...sending an update